snow/engine/common/tracker/peers.go

```diff
@@ -34,6 +35,12 @@ type Peers interface {
 	// SampleValidator returns a randomly selected connected validator. If there
 	// are no currently connected validators then it will return false.
 	SampleValidator() (ids.NodeID, bool)
+	// GetValidators returns the set of all validators
+	// known to this peer manager
+	GetValidators() set.Set[ids.NodeID]
+	// ConnectedValidators returns the set of all validators
+	// that are currently connected
+	ConnectedValidators() set.Set[ids.NodeID]
```

Comment:
It is a bit temping to just have a `DisconnectedValidators` here - but I think this interface is fine

Reply:
yea if we care about that performance gain here we can add it.